ui/helpers/utils/util.js

```diff
@@ -621,3 +621,7 @@ export const getNetworkNameFromProviderType = (providerName) => {
   }
   return providerName;
 };
+
+export const isAbleToExportAccount = (keyringType = '') => {
```

Comment:
Maybe add some JSDoc.

Reply:
Just added in commit d9c01765203c15f163465c62559d658151ce3942